quesma/table_resolver/table_resolver_dummy.go

```diff
@@ -0,0 +1,39 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+package table_resolver
+
+import (
+	mux "quesma_v2/core"
+)
+
+// DummyTableResolver is a dummy implementation of TableResolver to satisfy the QueryRunner and make it be compatible with the v2 api
+// thanks to this we can reuse the existing QueryRunner implementation without any changes.
+type DummyTableResolver struct{}
```

Comment:
I would consider naming it `EmptyTableResolver` as it better describes and we already use that name.

Reply:
I'll handle this in the next PR as this will have to also undergo a change related to common table handling 👍